src/components/Display.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Display({ result }) {
+  return (
+    <div className="display">
+      {result}
+    </div>
+  );
+}
+
+Display.defaultProps = {
+  result: 0,
+};
+
+Display.propTypes = {
+  result: PropTypes.string,
```

Comment:
- The default value of `result` should be `"0"` because it's proptype is a `string`

Reply:
Done!